test/data/testGeoOpt.py

```diff
@@ -180,6 +180,24 @@ def testoptstatus(self):
             self.assertEqual(self.data.optstatus[i], self.data.OPT_UNKNOWN)
         self.assertEqual(self.data.optstatus[-1], self.data.OPT_DONE)
 
+    @skipForParser('ADF', 'Not implemented yes')

```

Comment:
Looks like this block of skips just needs the comment to be changed from 'yes' -> 'yet'